src/platform/plugins/shared/saved_search/common/types.ts

```diff
@@ -46,14 +46,14 @@ export type VisContextUnmapped =
 /** @internal **/
 export interface SavedSearchAttributes {
```

Comment:
Should most of these props exist at all anymore if we're dropping all except `title`, `description`, and `tabs`? Would remove them completely help us catch potential issues via type errors?